utils/Misc.py

```diff
@@ -0,0 +1,34 @@
+'''
```

Comment:
It looks to me that these functions are not used in the repository. I would not commit this file